db/migrate/20160224155909_add_registered_at_to_subscription_facet.rb

```diff
@@ -0,0 +1,9 @@
+class AddRegisteredAtToSubscriptionFacet < ActiveRecord::Migration
+  def up
```

Comment:
I'd rather be explicit and not use this magic.  :mask: 

Readability is more important that being concise and I don't want to get into the habit of assuming rails will do certain things for me (or having to research whether or not it will when there is a perfectly good way to do something already).